src/murmur/Server.cpp

```diff
@@ -1473,7 +1473,7 @@ void Server::encrypted() {
 
 	QList< QSslCertificate > certs = uSource->peerCertificateChain();
 	if (!certs.isEmpty()) {
-		const QSslCertificate &cert = certs.last();
+		const QSslCertificate &cert = certs.first();
```

Comment:
```suggestion
		// Get the client's immediate SSL certificate
		const QSslCertificate &cert = certs.first();
```

Btw.: Is this the client's certificate that represents a client's identity? Or is this a separate certificate used for the TLS connection? Or is the "identity" in fact immediately used for the TLS connection?

Reply:
The identity is immediately used for the TLS connection 😄

Reply:
Okay that is good to know :sweat_smile: